src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -154,14 +176,47 @@ export default Vue.extend<{}, {}, Computed, Props>({
                         displayRows[key] = {
                             areaLabel,
                             areaHierarchy,
-                            filterLabels,
-                            indicatorValues: {}
+                            ...filterLabels,
                         }
                     }
-                    displayRows[key].indicatorValues[current.indicatorMeta.indicator] = current.value;
+                    displayRows[key][current.indicatorMeta.indicator] = current.value;
```

Comment:
We talked about prefixing the filter and indicator keys to avoid possible duplication (unlikely but possible). Did you decide against that?
Also maybe an issue when merge with your other ticket adding the upper and lower error columns..

Reply:
Because of the way the fields are generated for the table there isn't much of an issue of the indicators or filters being mislabelled -- unless you think there is a chance an indicator and a filter could have the same name? With regard to the error columns,  I took the liberty of adding them here. I prefixed them with the name of the indicator they belong to.

Reply:
PS. the uncertainty range format I've chosen has been approved by Jeff